src/NJsonSchema.CodeGeneration.CSharp/CSharpGeneratorSettings.cs

```diff
@@ -121,9 +121,12 @@ public CSharpGeneratorSettings()
         /// <summary>Gets or sets a value indicating whether to use preserve references handling (All) in the JSON serializer (default: false).</summary>
         public bool HandleReferences { get; set; }
 
-        /// <summary>Gets or sets the name of a static method which is called to transform the JsonSerializerSettings used in the generated ToJson()/FromJson() methods (default: null).</summary>
+        /// <summary>Gets or sets the name of a static method which is called to transform the JsonSerializerSettings (Newtonsoft.Json) used in the generated ToJson()/FromJson() methods (default: null).</summary>
         public string JsonSerializerSettingsTransformationMethod { get; set; }
```

Comment:
Maybe we should only use this method and use it for both cases just to avoid having mutual-exclusive options and avoiding adding more options... downside is the naming but what do you think?

Reply:
I think we can use it property for both json libraries. Just will rename it to `JsonSerializerSettingsOrOptionsTransformationMethod`. Do you agree?